cluster/examples/kubernetes/ozone/install-operator-rbac.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.ozone.rook.io
+spec:
+  group: ozone.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
+  scope: Namespaced
+  version: v1alpha1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-ozone-system
```

Comment:
I'd suggest using the `rook-ozone` namespace as the default instead of using a `system` suffix. At least we've moved away from that for the ceph operator.

Reply:
thx, done.